config/samples/core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml

```diff
@@ -0,0 +1,216 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack-network-isolation
+spec:
+  secret: osp-secret
+  storageClass: local-storage
+  cinder:
+    template:
+      cinderAPI:
+        containerImage: quay.io/tripleozedcentos9/openstack-cinder-api:current-tripleo
+        externalEndpoints:
+        - endpoint: internal
+          ipAddressPool: internalapi
+          loadBalancerIPs:
+          - 172.17.0.80
+      cinderBackup:
+        containerImage: quay.io/tripleozedcentos9/openstack-cinder-backup:current-tripleo
+        customServiceConfig: |
+          [DEFAULT]
+          backup_driver = cinder.backup.drivers.ceph.CephBackupDriver
+          backup_ceph_pool = backups
+          backup_ceph_user = openstack
+        networkAttachments:
+        - storage
+      cinderScheduler:
+        containerImage: quay.io/tripleozedcentos9/openstack-cinder-scheduler:current-tripleo
+      cinderVolumes:
+        volume1:
+          containerImage: quay.io/tripleozedcentos9/openstack-cinder-volume:current-tripleo
+          customServiceConfig: |
+            [DEFAULT]
+            enabled_backends=ceph
+            [ceph]
+            volume_backend_name=ceph
+            volume_driver=cinder.volume.drivers.rbd.RBDDriver
+            rbd_ceph_conf=/etc/ceph/ceph.conf
+            rbd_user=openstack
+            rbd_pool=volumes
+            rbd_flatten_volume_from_snapshot=False
+            rbd_secret_uuid=_FSID_
+          networkAttachments:
+          - storage
+          replicas: 0 # backend needs to be configured
+      extraMounts:
```

Comment:
@stuggi is there a particular reason to define these `extraMounts` here instead of having them in the top-level `spec` ?
If you define them at `openstack-operator` level you can use the same syntax to propagate the volumes to both `Cinder` and `Glance` (but you define them only once):

```
   - propagation:
              - CinderVolume
              - CinderBackup
              - GlanceAPI
```

Reply:
@fmount ah no, just forgot about the top-level one. will change it, thanks!

Reply:
Thanks, +1 on my side, we can merge this sample as long as it's ok for the other storage folks!